### ISSUE TITLE
Where is the Loss funtion

### ISSUE THREAD

**Issue by DonMuv** [HeatherJiaZG/SuperGlue-pytorch#18]
I can't find the loss function

**Comment by Shuhul24** [comment]
@DonMuv can you please let me know where have you figured out the loss function or was it intrinsically defined somewhere while training the model?
